python_scripts/03_categorical_pipeline_ex_02.py

```diff
@@ -101,3 +101,20 @@
 
 # %%
 # Write your code here.
+
+# %% [markdown]
```

Comment:
Probably this summary table should only be present in the solution and not the exercise.

In other words the exercise is an example which gives intuitions and kind of builds up towards the summary table.